modules/ossm-mirror-repository.adoc

```diff
@@ -0,0 +1,97 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/preparing-ossm-installation.adoc
+
+[id="ossm-mirror-repository_{context}"]
+= Mirroring the {product-title} image repository
+
+Mirror the {product-title} image repository to use during cluster installation
+or upgrade.
+
+.Prerequisites
+
+* You configured a mirror registry to use in your restricted network and
+can access the certificate and credentials that you configured.
+* You downloaded the pull secret from the
+link:https://cloud.redhat.com/openshift/install[OpenShift Infrastructure Providers]
+page and modified it to include authentication to your mirror repository.
+
+.Procedure
+
+Complete the following steps on the bastion host:
+
+. Review the
+link:https://access.redhat.com/downloads/content/290/[{product-title} downloads page]
+to determine the version of {product-title} that you want to install.
+
+. Set the required environment variables:
++
+----
+$ export OCP_RELEASE=<release_version> <1>
+$ export LOCAL_REGISTRY='<local_registry_host_name>:<local_registry_host_port>' <2>
+$ export LOCAL_REPOSITORY='<repository_name>' <3>
+$ export PRODUCT_REPO='openshift-release-dev' <4>
+$ export LOCAL_SECRET_JSON='<path_to_pull_secret>' <5>
+$ export RELEASE_NAME="ocp-release" <6>
+----
+<1> For `<release_version>`, specify the version number of {product-title} to
```

Comment:
@tvieira, @ yxun, could you please help Neal figure out what values for these environment variables should be for service mesh?

Reply:
Correct me if I am wrong. We didn't use any of the environment variables above from testing. Is that a question for our release side or errata related settings ?

Reply:
@yxun Are you referring to those environment variables when you were testing a disconnected install?

Reply:
@neal-timpe @JStickler we haven't used disconnected environment yet, I haven't seen those variables before. I will investigate and test.

Reply:
Had @gbaufake been testing this?